Modules/_elementtree.c

```diff
@@ -1810,12 +1812,19 @@ element_subscr(PyObject *op, PyObject *item)
         size_t cur;
         PyObject* list;
 
-        if (!self->extra)
-            return PyList_New(0);
-
         if (PySlice_Unpack(item, &start, &stop, &step) < 0) {
             return NULL;
         }
+
+        // Even if PySlice_Unpack() may clear 'self->extra',
+        // using a slice should not raise an IndexError, so
+        // we do not need to ensure 'extra' to exist.
+        //
+        // See https://github.com/python/cpython/issues/143200.
+        if (self->extra == NULL) {
+            return PyList_New(0);
+        }
+
```

Comment:
```suggestion
```

The check is directly related to the next line.

Reply:
The suggestion is about removing a blank line right?